tools/pyi/gen_pyi.py

```diff
@@ -679,10 +489,13 @@ def gen_pyi(declarations_path, out):
     for name in simple_conversions:
         unsorted_tensor_method_hints[name].append('def {}(self) -> Tensor: ...'.format(name))
 
-    tensor_method_declarations = get_py_variable_methods(declarations)
-    for name in sorted(tensor_method_declarations.keys()):
-        unsorted_tensor_method_hints[name] += \
-            generate_type_hints(name, tensor_method_declarations[name], namedtuples, is_tensor=True)
+    # pyi tensor methods don't currently include deprecated signatures for some reason
```

Comment:
Well, it's probably an implementation accident: deprecated signatures have to be read separately from deprecated.yaml, so if you forget to read in that file, ofc you're not going to generate signatures for them. This is probably wrong.

Reply:
sounds good, that'll be on the list of things to fix in a follow-up PR